docs/scripts/utils.py

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
can you provide a screenshot of exemplary results?

Reply:
Will do. Currently fixing some bugs.

Reply:
![image](https://github.com/emdgroup/baybe/assets/84016733/2fac01e5-f0aa-46a3-acaa-5cf33f2602e1)
![image](https://github.com/emdgroup/baybe/assets/84016733/320da883-4d0c-42fc-8b6d-85e4eb2585ef)
![image](https://github.com/emdgroup/baybe/assets/84016733/1518ec59-52eb-4622-9860-113552bde2c1)
<img width="963" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/ea353715-2ad9-4654-9c6e-29bc264eb465">
Here are some examples. Happy to provide more.

Reply:
@Scienfitz Once you resolve this I will merge both this and #118

Reply:
resolved est